MESC_Common/Src/MESCcli.c

```diff
@@ -896,7 +896,7 @@ static void cli_reply_begin( void )
     cli_reply_buffer_offset = 0;
 }
 
-static void cli_reply_close( void )
+void cli_reply_close( void )
```

Comment:
Not clear why this has been changed?

Reply:
Need to change it back. Something I did during debug.